op-geth.yaml

```
@@ -0,0 +1,129 @@
+package:
+  name: op-geth
+  version: "1.101411.6"
+  epoch: 0
+  description: The go-ethereum command line interface
+  copyright:
+    - license: LGPL-3.0-or-later
```

There are two licenses:

```suggestion
    - license: LGPL-3.0-or-later
    - license: GPL-3.0-or-later
```